source/documentation/doc-Release_Notes/chap-Certificates.adoc

```diff
@@ -0,0 +1,13 @@
+:_content-type: ASSEMBLY
+[id='sect-Certificates']
+= Managing certificates
+
+[role="_abstract"]
+{virt-product-fullname} {engine-name} uses certificates to enable encrypted communications. These {virt-product-shortname} certificates follow a standard 398 day lifetime and *must* be renewed once per year.
+
+[WARNING]
+====
+Do _NOT_ let certificates expire because once expired, the environment becomes non-operational and recovery is an error prone and time consuming process.
```

Comment:
Suggestion: 
Do _NOT_ let certificates expire.  If they expire, the environment becomes non-operational and recovery is an error prone and time consuming process.

Reply:
Changed.